src/Raids/AntorusTheBurningThrone/EonarLifebinder.js

```diff
@@ -8,6 +8,7 @@ export default {
   headshot: Headshot,
   fight: {
     vantusRuneBuffId: 250150,
+    noDowntimeSuggestion: true,
```

Comment:
How about `disableDowntimeStatistic` and `disableDowntimeSuggestion`?

I think by default it might be more desirable to just do `disableDowntimeSuggestion` and leave the statistic on so you can at least compare your own downtime with that of other players to find reasons why your performance is lower than that of others. What do you think?

Reply:
The problem is your down time is completely dictated by how high or low your raids dps is. If the group is low on damage, then there is less downtime between packs,  if your groups damage is high then the is a lot of downtime waiting for spawns.

But for healing maybe the downtime stat is still useful?

Reply:
Yeah and the personal downtime stat may be useless by itself. But if you're comparing yourself to other players with their (better) reports, the downtime stat might reveal that you and your raid - even if it's outside your own control - have more downtime. The reason for that might be due to personally being slow to cast things, move platforms or your raid being slow. I think having the statistic makes it easier to see.

If you're opposed to the statistic how about showing an alert box in it / apart of it that explains it's less accurate?